apps/vscode/command_client/command_client.py

```diff
@@ -153,7 +144,7 @@ def run_vscode_command(
         if args or return_command_output:
             raise Exception("Must use command-server extension for advanced commands")
         print("Communication dir not found; falling back to command palette")
-        run_vscode_command_by_command_palette(command_id)
+        actions.user.command_client_fallback(command_id)
```

Comment:
The idea would be to throw an exception here

Reply:
Done